authlib/specs/rfc7009/__init__.py

```diff
@@ -14,3 +14,4 @@
 from .parameters import prepare_revoke_token_request
 from .errors import UnsupportedTokenTypeError
 from .revocation import RevocationEndpoint
+from .registry import *
```

Comment:
from .registry import OAUTH_TOKEN_TYPE_HINTS

Reply:
I'd rather not make any changes in revocation spec. You can add a `SUPPORTED_TOKEN_TYPES` in IntrospectionEndpoint itself. In this way, people can custom the `SUPPORTED_TOKEN_TYPES` with a subclass.